lib/helpers.js

```diff
@@ -90,6 +93,19 @@ async function awaitRequest (promise) {
   return result;
 }
 
+async function parseIgnoreFile (path) {
+  try {
+    const file = await readfile(path);
+    return parse(file);
+  } catch (e) {
+    return [];
```

Comment:
Not following why this needs to return an empty array in case of error, other than that seems good.

Reply:
i don't think we can assume that a user is using git, so if they are not, we don't want to just fail, and the empty array plays into the "array merge" in the project-archiver

Reply:
thanks for the information :+1: